modules/apps/headless/headless-cms/headless-cms-impl/src/main/java/com/liferay/headless/cms/internal/resource/v1_0/BulkActionResourceImpl.java

```diff
@@ -18,4 +57,289 @@
 	scope = ServiceScope.PROTOTYPE, service = BulkActionResource.class
 )
 public class BulkActionResourceImpl extends BaseBulkActionResourceImpl {
+
+	@Override
+	public EntityModel getEntityModel(MultivaluedMap multivaluedMap)
+		throws Exception {
+
+		return _entityModel;
+	}
+
+	@Override
+	public BulkActionTask postBulkAction(
+			String search, Filter filter, BulkAction bulkAction)
+		throws Exception {
+
+		if (BulkAction.Type.DELETE_BULK_ACTION.equals(bulkAction.getType())) {
+			return _executeDeleteBulkAction(
+				bulkAction,
+				_getBulkActionItemMap(
+					bulkAction.getBulkActionItems(), filter, search,
+					GetterUtil.getBoolean(bulkAction.getSelectAll())));
+		}
+
+		throw new UnsupportedOperationException();
+	}
+
+	private BulkActionTask _addBulkActionTask(String type) throws Exception {
+		ObjectEntry bulkActionTask = _objectEntryLocalService.addObjectEntry(
```

Comment:
I think it's better just objectEntry as in this case we have also the type `BulkActionTask` and it's not the type of this variable